NAMESPACE

```diff
@@ -2,8 +2,12 @@
 
 S3method(summary_tbl_dea,Farrell)
 S3method(summary_tbl_dea,numeric)
+export(compute_efficiency)
 export(compute_scale_efficiency)
+export(compute_slack)
+export(compute_super_efficiency)
 export(create_matrix)
+export(get_peers)
```

Comment:
We do not need to export all functions at this stage, but for the analysis report that the user can download to reproduce the results, we need an exported function that returns the DEA model, slack and super efficiency.